test/90_1_prm/data/prototype_list.json

```diff
@@ -29,6 +29,8 @@
         ["Warehouse", "90.1-2013", "ASHRAE 169-2013-2A", []]
     ],
     "hvac_baseline": [
+        ["MediumOffice", "90.1-2013", "ASHRAE 169-2013-2A", [["remove_transformer", []], ["return_relief_fan", []]]],
```

Comment:
New test case that verifies that the baseline model includes return/relief fan, the baseline does as well, and that the baseline fan power is distributed to all three fans types similarly as the proposed model.